assets/js/textarea-image-uploader.js

```diff
@@ -12,5 +12,7 @@ export function setupImageUploader(selector) {
     },
   }
 
-  new Shubox(selector, shuboxOptions);
+  if ('undefined' !== typeof(Shubox)) {
```

Comment:
I'd be very happy to consider alternatives to this way of dealing with the issue that `Shubox` is undefined in the test and development environments. I'm not super happy with this.

Reply:
Perhaps get a review from someone in #javascript?

Reply:
Good idea.

Reply:
In my experiences, I've always seen this flipped, e.g. `typeof(Shubox) !== "undefined"`

Reply:
@MattMSumner `Shubox` gets defined by a [script we inject in the layout template](https://github.com/thoughtbot/constable/blob/master/lib/constable_web/templates/layout/app.html.eex#L14). I think it is defined in development (I was incorrect on that account) but it is not defined in the test because we use a dummy url (in https://github.com/thoughtbot/constable/blob/master/config/test.exs#L15)

@joshuaclayton that's helpful. But you think use `typeof(Shubox)` is okay instead of creating a null object or something like that?

Reply:
Thanks for the feedback. I flipped the conditional in 4bee967445568af337167fdfa4d42d549d549af5

Reply:
@germsvel my guess is that the page would still work without the JS addition; is that correct? If so, I'd avoid a shim/null object and allow the browser to handle it the old-fashioned way.

If there *is* some sensible fallback behavior (that's still handled by JS), a null object seems like a reasonable approach, but perhaps better suited for a separate PR anyway. Thoughts?

Reply:
Yeah, that makes sense. The page works totally fine without `Shubox` being there, so there's no harm in leaving it like it is. And you're right, if there is some fallback behavior, I can introduce it in a separate PR. 

Thanks for the feedback!

Reply:
What is Shubox? Is it this? https://shubox.io

Reply:
Yeah, that's the one. A former thoughtbotter's side project.